operations/iuf/IUF.md

```diff
@@ -604,7 +606,8 @@ products. `product_vars.yaml` is provided by HPE and the values are intended as
 
 Site variables, typically specified in a `site_vars.yaml` file, allow the administrator to override values provided by recipe
 variables, including global default entries and product-specific entries. HPE does not provide a `site_vars.yaml` file as it is
-strictly for site use cases.
+strictly for site use cases. See the text at the top of the HPE-provided `product_vars.yaml` file for details on which override
+values can be specified in `site_vars.yaml`.
```

Comment:
it occurs to me that it might be good if these could be hot links to a page about the file mentioned.  that is to say, we have a "product vars" page, and then when we mention product vars, we link to it.

Reply:
You are suggesting a separate docs page for product vars specifically?  What would be in it, as opposed to what is in this section?  Or would it be an example product_vars.yaml file?  Just trying to better understand the comment.

Reply:
Will work with John offline on this RFE.